src/CmsManager/DecoratorStrategy.php

```diff
@@ -57,12 +57,14 @@ public function isDecorable(Request $request, $requestType, Response $response)
             return false;
         }
 
-        if (true === $response->headers->get('x-sonata-page-not-decorable', false)) {
+        // TODO: drop support for `true` value when bumping symfony/http-foundation dependency to >= 3.4.31
+        if (\in_array($response->headers->get('x-sonata-page-not-decorable', '0'), [true, '1'], true)) {
```

Comment:
```suggestion
        if (true === (bool) $response->headers->get('x-sonata-page-not-decorable', false)) {
```
Is more understandable

Reply:
More understandable but casting is less strict as it allows us to support "out of scope" HttpFoundation implementation (for example if userland extends the headers bag and break the contract) + the default value passed to `get` must be a string.

Reply:
Why not use `getBoolean` here?

Reply:
`getBoolean` doesn't exist in `ResponseHeaderBag`

Reply:
Oh right, I got confused with `ParameterBag`, nevermind

Reply:
It was a great suggestion actually, I was disappointed too 😅